src/views/ReturnDetails.vue

```diff
@@ -11,7 +11,7 @@
       <main>
         <div class="doc-id">
           <ion-item lines="none">
-            <h1>{{ current.shopifyOrderName ? current.shopifyOrderName : current.hcOrderId }}</h1>
+            <h1> {{ current.shopifyOrderName ? current.shopifyOrderName : current.hcOrderId ? current.hcOrderId : current.externalId ? current.externalId : 'Return' }}</h1>
```

Comment:
We need to make the static text internationalize like the one done for `Scan barcodes to receive them` text on line 28.

Also can you make the static text same as the one displayed in title `Return Details`